modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -7,23 +7,24 @@ import ClayBadge from '@clayui/badge';
 import ClayButton from '@clayui/button';
 import ClayDropDown from '@clayui/drop-down';
 import ClayForm, {ClayInput} from '@clayui/form';
-import ClayLayout from '@clayui/layout';
 import ClayModal from '@clayui/modal';
 import {FrontendDataSet} from '@liferay/frontend-data-set-web';
 import classNames from 'classnames';
 import {fetch, navigate, openModal} from 'frontend-js-web';
-import fuzzy from 'fuzzy';
 import React, {useState} from 'react';
 
 import '../css/FDSEntries.scss';
 import {FDSViewType} from './FDSViews';
 import RequiredMark from './components/RequiredMark';
 import ValidationFeedback from './components/ValidationFeedback';
+import RESTApplicationDropdownItem from './components/rest/RESTApplicationDropdownItem';
+import RESTApplicationDropdownMenu from './components/rest/RESTApplicationDropdownMenu';
+import RESTEndpointDropdownMenu from './components/rest/RESTEndpointDropdownMenu';
+import RESTSchemaDropdownMenu from './components/rest/RESTSchemaDropdownMenu';
```

Comment:
I could have left these duplicated, but @chestofwonders will need these in selection filters anyways.

Reply:
Nice!